### DISCUSSION TITLE
How to redirect from server/api?

### DISCUSSION THREAD

**Discussion by mihail727** [nuxt/nuxt#17859] [Questions]
How to redirect from `@/server/api/logout.ts`
`res.writeHead(302, { Location: '/auth/login' })` --- doesnt work

**Comment by TheAlexLichter** [comment]
Have you tried sending `res.end()` afterwards?

```js
res.writeHead(302, { Location: '/your-location/' })
res.end()
```

**Reply by mihail727** [reply]
> Have you tried sending `res.end()` afterwards?
> 
> ```js
> res.writeHead(302, { Location: '/your-location/' })
> res.end()
> ```

yes
